### PR TITLE
Update config.go

### PR DESCRIPTION
**Pull request by guoguodan**
it maybe null point

### CODE REVIEW

File: cmd/yurthub/app/config/config.go [hunk 1]
```diff
@@ -164,6 +164,9 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 }
 
 func parseRemoteServers(serverAddr string) ([]*url.URL, error) {
+	if serverAddr == "" {
+		return make([]*url.URL, 0), fmt.Errorf("--server-addr should be set for hub agent")
```
Comment (Fei-Guo) [inline cmd/yurthub/app/config/config.go:168]: We have line 189 below which should protect the case that serverAddr is "".

Comment (rambohe-ch) [reply cmd/yurthub/app/config/config.go:168]: I think that line 189 will check resolved serverAddress, and this pr only check the parameter at the func entry. it's make sense.

Comment (rambohe-ch) [reply cmd/yurthub/app/config/config.go:168]: > We have line 189 below which should protect the case that serverAddr is "".

@Fei-Guo I think that line 189 will check resolved serverAddress, and this pr only check the parameter at the func entry. it's make sense.

**Pull request merged**